packages/solarwinds-apm/build.js

```diff
@@ -33,3 +35,33 @@ await fs.writeFile("src/version.ts", linted[0].output)
 
 await fs.mkdir("dist/commonjs", { recursive: true })
 await fs.cp("src/commonjs/package.json", "dist/commonjs/package.json")
+
+// Generate the web instrumentation bundle to target the Chrome/Safari/Firefox/Edge
+// versions which were the latest 1 year ago as a default
```

Comment:
Is 1 year the otel convention and/or what other custom distros are doing?

Reply:
I picked 1 year because it's in line with our support policy, it's actually pretty generous. The policy we've decided on for OTel JS is targeting ES2022 which is roughly equivalent to this at the moment, and by default most bundlers actually only support the latest 2 versions of major browsers which is way stricter.